validator/project_validator.go

```diff
@@ -619,7 +658,7 @@ func checkProjectFields(project *model.Project) ValidationErrors {
 // 1. A referenced task within a buildvariant task object exists in the set of project
 // tasks and is not referenced in the task group of the buildvariant.
 // 2. Any referenced distro exists within the current setting's distro directory
```

Comment:
Update this comment. I might remove some of the excessive comment description since it seems outdated and just summarize that `ensureReferentialIntegrity` checks that fields that reference other named things (e.g. task names, distro names, container names) are referring to valid names.